lib/grizzly/commands/command.ex

```diff
@@ -373,15 +373,19 @@ defmodule Grizzly.Commands.Command do
   end
 
   defp calculate_rssi_values(stats) do
-    average =
+    min_rssi =
       Keyword.get(stats, :rssi_hops, [])
-      |> Enum.reject(fn rssi ->
-        rssi in [:not_available, :max_power_saturated, :below_sensitivity]
-      end)
-      |> calculate_average_dbm()
+      |> Enum.filter(&is_number/1)
+      |> Enum.min(fn -> :not_available end)
+
+    bars =
+      case min_rssi do
+        v when is_number(v) -> calculate_bars(v)
+        _ -> :not_available
+      end
```

Comment:
Not blocking, but I would have looked into adding another `calculate_bars/1` implementation rather than have a mini `case` block here. I think that you only have to handle `:not_available` since anything not-a-number was filtered out.